content/sensu-go/6.5/operations/deploy-sensu/install-sensu.md

```diff
@@ -125,20 +125,30 @@ For details about intialization in Docker, see the [backend reference](../../../
 {{< language-toggle >}}
 
 {{< code Docker >}}
-# Replace `<username>` and `<password>` with the username and password
-# you want to use for your admin user credentials.
+# Use either username and password or API key for credentials.
+# To use username and password, replace <username> and <password> with
+# the username and password you want to use for your admin user credentials
+# and delete the API key line.
+# To use API key, replace <api_key> with the API key you want to use for
```

Comment:
@echlebek what if this is your first cluster? How do you init with an API key if you have never had a backend running yet?